src/components/form-create-account/form-create-account.js

```diff
@@ -0,0 +1,268 @@
+            options={lastPeriodInformationOptions}
+          />
+        </div>
+      </div>
+      <div className="columns">
+        <div className="column is-half">
+          <Select
+            label="Alegeți țara în care vă aflați"
+            selectProps={{
+              name: "country",
+              id: "country",
+              onChange: country_OnChange
+            }}
+            options={countriesList}
+          />
+        </div>
+        <div className="column is-half">
+          <Select
+            label="Alegeți orașul în care vă aflați"
+            selectProps={{
+              name: "city",
+              id: "city",
+              onChange: input_OnChange
+            }}
+            options={cities}
+          />
+        </div>
+      </div>
+      <div className="columns">
+        <div className="column">
+          <label className="checkbox">
+            <input
+              type="checkbox"
+              required
+              name="acceptConditions"
+              id="acceptConditions"
+              onChange={input_OnChange}
+            />
+            &nbsp;&nbsp;&nbsp; Prin această bifă îți exprimi acordul ca datele
+            furnizate de tine prin acest formular să fie procesate exclusiv in
+            scopul de a te pune în legătură cu un specialist care să te ajute cu
+            problema pentru care cauți soluție.{" "}
+            <a href="#" onClick="return false;">
+              Aici puteți găsi regulamentul nostru cu privire la prelucrarea
+              datelor cu caracter personal.
+            </a>
+          </label>
+        </div>
+      </div>
+    </div>
+  );
+};
+
+FormCreateAccount.propTypes = {
+  objectChanged: PropTypes.func,
+  ageOptions: PropTypes.arrayOf(
+    PropTypes.shape({
+      text: PropTypes.string.isRequired,
+      value: PropTypes.string.isRequired,
+      selected: PropTypes.bool
+    })
+  ),
+  genderOptions: PropTypes.arrayOf(
+    PropTypes.shape({
+      text: PropTypes.string.isRequired,
+      value: PropTypes.string.isRequired,
+      selected: PropTypes.bool
+    })
+  ),
+  illnesesOptions: PropTypes.arrayOf(
+    PropTypes.shape({
+      text: PropTypes.string.isRequired,
+      value: PropTypes.string.isRequired,
+      selected: PropTypes.bool
+    })
+  ),
+  lastPeriodInformationOptions: PropTypes.arrayOf(
+    PropTypes.shape({
+      text: PropTypes.string.isRequired,
+      value: PropTypes.string.isRequired,
+      checked: PropTypes.bool
+    })
+  ),
+
+  countriesList: PropTypes.arrayOf(
+    PropTypes.shape({
+      text: PropTypes.string.isRequired,
+      value: PropTypes.string.isRequired,
+      checked: PropTypes.bool
+    })
+  ),
+  citiesList: PropTypes.arrayOf(
+    PropTypes.shape({
+      text: PropTypes.string.isRequired,
+      value: PropTypes.string.isRequired,
+      checked: PropTypes.bool,
+      countryCode: PropTypes.string.isRequired
+    })
+  )
+};
+
+FormCreateAccount.defaultProps = {
```

Comment:
Pentru a evita fisierele prea mari, constanta asta s-ar putea extrage intr-un alt fisier

Reply:
Extracted to constants in ...propTypes.js and ...defaultValus.js files